<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfPlotDockWidget</class>
 <widget class="QDockWidget" name="WkfPlotDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="wkf::PlotWidget" name="plotWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="ut::qt::AxisBoundsWidget" name="xAxisBounds">
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ut::qt::AxisBoundsWidget" name="yAxisBounds">
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearDataPushButton">
        <property name="text">
         <string>Clear Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>289</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::PlotWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>WkfPlotWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>ut::qt::AxisBoundsWidget</class>
   <extends>QGroupBox</extends>
   <header>UtQtAxisBoundsWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
